### PR TITLE
Fast scnn

### PR DESCRIPTION
**Pull request by LielinJiang**
Add fast scnn model.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ $ pip install -r requirements.txt
 * [如何训练ICNet](./turtorial/finetune_icnet.md)
 * [如何训练PSPNet](./turtorial/finetune_pspnet.md)
 * [如何训练HRNet](./turtorial/finetune_hrnet.md)
+* [如何训练Fast_scnn](./turtorial/finetune_fast_scnn.md)
```
Comment (qingqing01) [inline README.md]: Fast_scnn -> Fast-SCNN

File: turtorial/finetune_fast_scnn.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+python pdseg/train.py --use_gpu --cfg ./configs/fast_scnn_pet.yaml
+```
+
+## 六. 进行评估
+
+模型训练完成，使用下述命令启动评估
+
+```shell
+python pdseg/eval.py --use_gpu --cfg ./configs/fast_scnn_pet.yaml
+```
```
Comment (qingqing01) [inline turtorial/finetune_fast_scnn.md:110]: better give the benchmark, for example,  the speed comparison with other models, to tell users the value of this model

File: README.md (hunk 1 above)
Comment (LielinJiang) [reply README.md]: updated

File: turtorial/finetune_fast_scnn.md (hunk 2 above)
Comment (LielinJiang) [reply turtorial/finetune_fast_scnn.md:110]: added

**Comment by ZeyuChen** [comment]
LGTM

**Pull request merged**